include/swift/AST/ASTBridging.h

```diff
@@ -371,25 +378,67 @@ void BridgedDiagnostic_fixItReplace(BridgedDiagnostic cDiag,
 SWIFT_NAME("BridgedDiagnostic.finish(self:)")
 void BridgedDiagnostic_finish(BridgedDiagnostic cDiag);
 
+//===----------------------------------------------------------------------===//
+// MARK: DeclContexts
+//===----------------------------------------------------------------------===//
+
+SWIFT_NAME("getter:BridgedDeclContext.isLocalContext(self:)")
+bool BridgedDeclContext_isLocalContext(BridgedDeclContext cDeclContext);
+
+SWIFT_NAME("BridgedPatternBindingInitializer.create(declContext:)")
+BridgedPatternBindingInitializer
+BridgedPatternBindingInitializer_create(BridgedDeclContext cDeclContext);
+
+SWIFT_NAME("getter:BridgedPatternBindingInitializer.asDeclContext(self:)")
+BridgedDeclContext BridgedPatternBindingInitializer_asDeclContext(
+    BridgedPatternBindingInitializer cInit);
+
 //===----------------------------------------------------------------------===//
 // MARK: Decls
 //===----------------------------------------------------------------------===//
 
+enum ENUM_EXTENSIBILITY_ATTR(closed) BridgedStaticSpelling {
+  BridgedStaticSpellingNone,
+  BridgedStaticSpellingStatic,
+  BridgedStaticSpellingClass
+};
+
+enum ENUM_EXTENSIBILITY_ATTR(closed) BridgedAccessorKind {
+#define ACCESSOR(ID) BridgedAccessorKind##ID,
+#include "swift/AST/AccessorKinds.def"
+};
+
+struct BridgedAccessorRecord {
```

Comment:
I was thinking like literally bridging `AbstractStorageDecl::AccessorRecord` and make`AbstractStorageDecl::setAccessors` accept `AbstractStorageDecl::AccessorRecord`.
But considering the current Bridging facility, this might be simpler.

Reply:
Yeah that was my feeling too